packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -183,6 +189,14 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         return this.registry;
     }
 
+    public get IFluidRouter() {
```

Comment:
this will create a new object on every invokation. you can use a private Lazy<IFluidRouter> to generate only once and on demand